tests/test_pointing.py

```diff
@@ -0,0 +1,199 @@
+# Build up sequence of pointing offsets running linearly in x and y directions
+scan = np.linspace(-max_extent, max_extent, num_pointings // 2)
+offsets_along_x = np.c_[scan, np.zeros_like(scan)]
+offsets_along_y = np.c_[np.zeros_like(scan), scan]
+offsets = np.r_[offsets_along_y, offsets_along_x]
+
+# Creating list of antenna objects
+ANT_DESCRIPTIONS = ["""m000,
+        -30:42:39.8,
+        21:26:38.0,
+        1086.6,
+        15,
+        -8.264 -207 8.6 212.6 212.6 1,
+        0:04:20.6 0 0:01:14.2 0:02:58.5 0:00:05.1 0:00:00.4
+                          0:20:04.1 -0:00:34.5 0 0 -0:03:10.0 ,1.22""",
+                    """m001,
+        -30:42:39.8,
+        21:26:38.0,
+        1086.6,
+        15,
+        -8.264 -207 8.6 212.6 212.6 1,
+        0:04:15.6 0 0:01:09.2 0:01:58.5 0:00:05.1 0:00:00.4
+                          0:16:04.1 -0:00:34.5 0 0 -0:03:10.0 ,1.22"""]
+
+
+ants = [katpoint.Antenna(ant) for ant in ANT_DESCRIPTIONS]
+az_el_adjust = np.zeros((len(ants), 2))
+
+# Creating gains, numpy array shape (no.offsets, no.polarisations, no.antennas)
+just_gains = np.ones((len(offsets), len(POLS), len(ants)), dtype=np.complex64)
+
+
+def generate_bp_gains(offsets, ants, channel_freqs, pols, beam_center=(0.5, 0.5)):
+    """ Creating bp_gains, numpy array of shape:
+        (no.offsets, no.freqs, no.polarisations, no.antennas)"""
+    bp_gains = np.zeros(
+        (len(offsets),
+         len(channel_freqs),
+         len(pols),
+         len(ants)),
+        dtype=np.complex64)
+    for i in range(len(offsets)):
+        for f in range(len(channel_freqs)):
+            for pol in range(len(pols)):
+                ex_width = []
+                for a, ant in enumerate(ants):
+                    # expected widths for each frequency channel
+                    expected_width = pointing._voltage_beamwidths(
+                        ant.beamwidth, channel_freqs[f], ant.diameter)
+                    ex_width.append(expected_width)
+
+                gains = []
+                for k in ex_width:
+                    new_beam = pointing.BeamPatternFit(beam_center, k, 1.0)
+                    g = new_beam(x=offsets[i].T)
+                    cmplx = random.uniform(-1.5, 1.5)
+                    g = cmath.rect(g, cmplx)
+                    gains.append(np.array(g))
+                gains = np.array(gains).T
+                bp_gains[i][f][pol] = gains
+
+    return bp_gains
+
+
+bp_gains = generate_bp_gains(offsets, ants, channel_freqs, POLS)
+data_points = pointing.get_offset_gains(
+    bp_gains,
+    just_gains,
+    offsets,
+    ants,
+    TRACK_DURATION,
+    CENTER_FREQ,
+    BANDWIDTH,
+    N_CHANNELS,
+    POLS,
+    NUM_CHUNKS)
+beams = pointing.beam_fit(data_points, ants, NUM_CHUNKS)
+pointing_offsets = pointing.calc_pointing_offsets(
+    ants,
+    MIDDLE_TIME,
+    TEMPERATURE,
+    HUMIDITY,
+    PRESSURE,
+    beams,
+    target,
+    az_el_adjust)
+
+
+# Test that length of data_points equals the legnth of antenna list
```

Comment:
These comments above the tests are better implemented as docstrings in the test functions, like this
```python
def test_get_offset_gains_len():
    """Test that length of data_points equals the length of antenna list."""
```
Same goes for the others.